doc/source/reference/c-api.array.rst

```diff
@@ -3558,10 +3562,17 @@ Enumerated Types
 
 .. c:type:: NPY_SORTKIND
 
-    A special variable-type which can take on the values :c:data:`NPY_{KIND}`
-    where ``{KIND}`` is
+    A special variable-type which can take on the following values
```

Comment:
Since we are here, maybe we can add a (half) sentence repeating that it signals whichalgorithm is used for sorting. I know its pretty obvious though.